src/main/java/com/savvato/tribeapp/controllers/ReviewDecisionAPIController.java

```diff
@@ -12,24 +15,37 @@
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
+import java.util.List;
+import java.util.Optional;
+
 @RestController
 @Tag(name = "reviewer-decision", description = "Decisions made after reviewing an attribute")
 @RequestMapping("/api/reviewer-decision")
 public class ReviewDecisionAPIController {
   @Autowired ReviewDecisionService reviewDecisionService;
 
+  @Autowired ReviewDecisionReasonService reviewDecisionReasonService;
```

Comment:
Check the other services for context. Our autowired services are defined on two lines: the first @autowired, and the second the type and variable name of the service. For consistency, you should do that as well.